src/logtab.cpp

```diff
@@ -612,12 +618,49 @@ void LogTab::ChangePrevIndex()
     }
 }
 
+void NormalizeLogText(QString& logText)
+{
+    logText.replace(QRegularExpression("id: (-)?\\d+"), "id: N");
```

Comment:
Changed to use map. Using vector<tuple> would faster but this is more convenient and the list is short.

Sample:
```
start-test	i: N; n: N; test: ExtractActionsUnitTest::testCalcs
end-query	cols: 0; elapsed: N.N; protocol-id: N; query: SELECT TOP 1 * INTO [#Tableau_ID_Connect] FROM (SELECT 1 AS COL) AS CHECKTEMP; query-category: Capability; query-hash: N; rows: 0

```
Binary also available in internal TLV file share.